src/lib/structures/expr.mli

```diff
@@ -72,7 +72,25 @@ and quantified = private {
   (** The underlying formula. *)
 
   toplevel : bool;
-  (** Determine if the quantified formula is at the top level.
+  (** Determine if the quantified formula is at the top level of an asserted
+      formula.
+
+      An {e asserted formula} is a formula introduced by {e (assert ...)} or
+      generated by a function definition with {e (define-fun ...)}.
+
+      By {e top level}, we mean that the quantified formula is not
+      contained in another quantified formula, but the formula can be a
+      subformula.
```

Comment:
That is confusing. Isn't a formula contained in another formula a subformula? Maybe we can use some clearer wording?
For example:

```suggestion
      By {e top level}, we mean that the quantified formula is not
      a subformula of another quantified formula.
```
Should suffice IMO.